src/ballet/chacha20/test_fd_chacha20_rng.c

```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
```

Comment:
If you need file IO, it should be protected with `#if FD_HAS_HOSTED`, but I don't think this test actually needs file IO.

Reply:
Just removed `stdio.h` as no  IO is needed